mmv1/third_party/terraform/resources/resource_tags_location_tag_bindings.go.erb

```diff
@@ -101,7 +101,7 @@ func resourceTagsLocationTagBindingCreate(d *schema.ResourceData, meta interface
 	// identity fields and d.Id() before read
 	
 	var opRes map[string]interface{}
-	err = tagsLocationOperationWaitTimeWithResponse(
+	err = TagsOperationWaitTimeWithResponse(
```

Comment:
This looks like it's calling a different method now?

Reply:
Good catch. Thanks.